model/Concept.php

```diff
@@ -436,14 +436,14 @@ public function getMappingProperties(array $whitelist = null)
                             }
 
                             if ($response) {
-                                $ret[$prop]->addValue(new ConceptMappingPropertyValue($this->model, $this->vocab, $response, $this->resource, $prop), $this->clang);
```

Comment:
Funny, these `clang` parameters are not in the `addValue` signature.

Reply:
Yes, seems like the signature was changed some five years ago: https://github.com/NatLibFi/Skosmos/commit/2ba0a48174df87965e7785c8196ce2b039a85050#

Good catch!